### PR TITLE
First pass at filtering.

### PR DESCRIPTION
**Pull request by JackRyan1989**
Needed to update seed script so some programs come through. Also set has queue to True initially, I think we want folks to manually set the program to not have a queue that intuitively makes sense to me.

### CODE REVIEW

File: core/management/commands/seed.py [hunk 1]
```diff
@@ -239,6 +239,7 @@ def create_programs(output=True):
     for program, services in DEFAULT_PROGRAMS.items():
         p = Program()
         p.name = program
+        p.has_queue = random_bool()
```
Comment (MikeyManoguerra) [inline core/management/commands/seed.py:242]: Even though we aren't using the queue right now, which programs have queues should not be random. 
We definitely do not want SEP to have a queue at the moment. 

Create an list of strings const called HAS_QUEUE, with a few program names "STEP" , "LEGAL" and a couple more. then if the program is in the list while seeding, make the has queue flag true

File: frontend/src/views/AllQueues.js [hunk 2]
```diff
@@ -52,9 +52,17 @@ const AllQueues = observer(() => {
     api
       .getPrograms()
       .then(({ data }) => {
```
Comment (MikeyManoguerra) [inline frontend/src/views/AllQueues.js:54]: we should run the filter before we make the requests to the queue endpoint. That endpoint is relatively expensive, and nothing will come back anyway.

try using Array.filter method, itll be a little more clear what is going on

**Review by MikeyManoguerra — CHANGES_REQUESTED** [review]
just a couple small things

File: frontend/src/views/AllQueues.js (hunk 2 above)
Comment (JackRyan1989) [reply frontend/src/views/AllQueues.js:54]: So do you think where I'm implementing the filter is ok @MikeyManoguerra ?

Comment (MikeyManoguerra) [reply frontend/src/views/AllQueues.js:54]: Yes, If its something that a more common operation we could seperate it out into a fn, but this is the only place we are doing it so far

**Review by MikeyManoguerra — APPROVED** [review]

**Pull request merged**